browser/src/Plugins/Plugin.ts

```diff
@@ -51,8 +61,9 @@ export class Plugin {
 
         try {
             vm.runInNewContext(
-                `debugger; const pluginEntryPoint = require('${moduleEntryPoint}').activate; if (!pluginEntryPoint) { console.warn('No activate method found for: ${moduleEntryPoint}'); } else { pluginEntryPoint(Oni); } `,
+                `debugger; const pluginEntryPoint = require('${moduleEntryPoint}').activate; if (!pluginEntryPoint) { console.warn('No activate method found for: ${moduleEntryPoint}'); } else { pluginContainer._instance = pluginEntryPoint(Oni); } `,
```

Comment:
Cool, I like this a lot! Some interesting possibilities this opens up... like library plugins that provide common functionality, or plugins that supply functions that you can use with `input.bind`, etc. Great way to get access to the plugin to test, too.